### PR TITLE
GitHub default fix

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Fixed an issue where is some case get-branch would've failed if no author appeared in the response

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Link to the failing unit test replicates the the get-branch command:
https://app.circleci.com/pipelines/github/demisto/content/126581/workflows/8f42463e-c6cc-4f84-8e80-75bb3d2ea345/jobs/447173

File: Packs/GitHub/ReleaseNotes/1_3_10.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### GitHub
+- Fixed an issue where the **GitHub-get-branch** command failed to parse response in case there was no PR author.
```
Comment (yaakovi) [inline Packs/GitHub/ReleaseNotes/1_3_10.md:4]: no PR or no PR author?

Comment (ShahafBenYakir) [reply Packs/GitHub/ReleaseNotes/1_3_10.md:4]: no PR author

Comment (yaakovi) [reply Packs/GitHub/ReleaseNotes/1_3_10.md]: how is that possible? deleted user?

**Review by yaakovi — APPROVED** [review]

**Review by adi88d — APPROVED** [review]

File: Packs/GitHub/ReleaseNotes/1_3_10.md (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/GitHub/ReleaseNotes/1_3_10.md]: Also might be. Tho I think in this case it's not related it is this branch https://github.com/demisto/content-private/pull/473

**Pull request merged**